packages/components/src/EditableText/EditableText.component.js

```diff
@@ -80,19 +82,20 @@ export function EditableTextComponent({ editMode, loading, inProgress, ...rest }
 EditableTextComponent.displayName = 'EditableText';
 
 EditableTextComponent.propTypes = {
-	text: PropTypes.string.isRequired,
+	componentClass: PropTypes.string,
```

Comment:
(non-blocking comment) Personally I find componentClass misleading and would rather see ElementType or something (with leading capital too). I know bootstrap uses it, but this component is our own and that way we don't confuse it with a bootstrap component anyway. What do you think about it?

Reply:
I find it more confusing to have 2 different props. So user have to know if it's something from ui or from bootstrap to know which one to use.
Having the same one remove this possible confusion IMO

Reply:
I meant the name only, not necessarily splitting it into 2 props. `"h1"` is not a class, so the name made no sense to me

Reply:
So you still have to know if it's from bootstrap so you can go to bootstrap to see what it actually means :)